bidder/bidder.go

```diff
@@ -0,0 +1,181 @@
+package bidder
+
+import (
+	"context"
+	"sync"
+	"time"
+
+	"github.com/BOPR/bazooka"
+	"github.com/BOPR/config"
+	"github.com/BOPR/core"
+	"github.com/BOPR/db"
+	"github.com/BOPR/log"
+	ethCmn "github.com/ethereum/go-ethereum/common"
+)
+
+var (
+	BiddingService = "bidder"
+)
+
+type CoordinatorInfo struct {
+	DepositAmount uint64
+	Address       ethCmn.Address
+}
+
+type SlotInfo struct {
+	CurrentSlot uint64
+}
+
+type Bidder struct {
+	// Base service
+	core.BaseService
+
+	// DB instance
+	DB db.DB
+
+	bz bazooka.Bazooka
+
+	cfg config.Configuration
+
+	bidderInfo CoordinatorInfo
+
+	// header listener subscription
+	cancelBidding context.CancelFunc
+
+	// wait group
+	wg sync.WaitGroup
+}
+
+// NewBidder returns new aggregator object
+func NewBidder(cfg config.Configuration) *Bidder {
+	// create logger
+	logger := log.Logger.With("module", BiddingService)
+	bi := &Bidder{}
+	bi.BaseService = *core.NewBaseService(logger, BiddingService, bi)
+	DB, err := db.NewDB(cfg)
```

Comment:
do we need db for Bidder? Looks like it is unused.

Reply:
ah yes, need to remove, good eye!